### PR TITLE
Pr/14 newsmarmour

### PR DESCRIPTION
**Pull request by Blogaugis**
Adds Mark 1, 2 and heavy PA variants. Currently not build-able, more as reference, and need images.
More hireling armors added.
Added more TODO comments.

### CODE REVIEW

**Review by EttyKitty — COMMENTED** [review]
@Blogaugis. I would say that I'm okay with the light power armour for hirelings and some fixes that you've made, where some code chunks miss existing armour listed. Minus Cata armour, as it has no sprite as well.

MK1, MK2, Heavy PA - are something that I'm hesitant, as they don't have sprites and generally questionable at this point.

**Comment by EttyKitty** [comment]
Actually, light power armour is also added in this PR #27.

**Comment by Blogaugis** [comment]
> Actually, light power armour is also added in this PR #27.

Part of the issue that I wanted to change is that it depends on the work in other places. Atomizing the PRs may not be the best option at the moment.
Should I also merge this PR with the hireling fixes?

**Comment by EttyKitty** [comment]
> Should I also merge this PR with the hireling fixes?

I would say yes, but remove parts where you add heavy PA, mk1 and mk2. And also, probably remove Cata term armour where you've added it, as it has no sprite and is kinda hidden atm.

**Comment by Blogaugis** [comment]
> I would say yes, but remove parts where you add heavy PA, mk1 and mk2. And also, probably remove Cata term armour where you've added it, as it has no sprite and is kinda hidden atm.

Eh, the new added armours are also hidden. You can't buy them or otherwise acquire them. They are more as a reference. Frankly, if we are really removing them, then I think it is better to just close this PR and focus on #27 instead.
So, should I close this?

File: scripts/scr_weapon/scr_weapon.gml [hunk 1]
```diff
@@ -2011,6 +2062,53 @@ global.gear = {
...
+		},
+		"Ranger Armour":{
+			"abbreviation": "RngrArm",
+			"description": "This armour is used by eldar rangers.",
+			"armour_value": {
+				"standard": 25,
+				"master_crafted": 27,
+				"artifact": 30
+			},
+		},
```
Comment (EttyKitty) [inline scripts/scr_weapon/scr_weapon.gml:2082]: You may want to transfer this into that pr then.

File: scripts/scr_weapon/scr_weapon.gml [hunk 2]
```diff
@@ -2011,6 +2062,53 @@ global.gear = {
...
+			"description": "Mismatched basic armour used by ork forces"
+		},
+		"Fire Warrior Armour": {
+			"abbreviation": "FWarArm",
+			"description": "This armour is used by T'au fire warriors.",
+			"armour_value": { // TODO - needs rebalancing
+				"standard": 20,
+				"master_crafted": 22,
+				"artifact": 25
+			},
```
Comment (EttyKitty) [inline scripts/scr_weapon/scr_weapon.gml:2109]: And this:

File: scripts/scr_weapon/scr_weapon.gml (hunk 1 above)
Comment (Blogaugis) [reply scripts/scr_weapon/scr_weapon.gml:2082]: Alright.

File: scripts/scr_weapon/scr_weapon.gml (hunk 2 above)
Comment (Blogaugis) [reply scripts/scr_weapon/scr_weapon.gml:2109]: On it.

**Comment by Blogaugis** [comment]
As mentioned earlier, new armours are not going to be added at this time.

**Pull request closed without merging**